alerts/get_watchlist.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, QueryStringMatch
+import requests
+import json
+import logging
+from requests_jwt import JWTAuth
+import sys
+
+logger = logging.getLogger()
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+
+class AlertWatchList(AlertTask):
+    def main(self):
+        global watchterm
```

Comment:
Instead of using global variables here, you can use an instance variable to keep track of the watchterm. For example `self.watchterm = 'someusername'`

Reply:
Will try that, thank you!